redash/query_runner/db2.py

```diff
@@ -0,0 +1,147 @@
+import os
+import json
+import logging
+import select
+
+import ibm_db_dbi
```

Comment:
Please follow a pattern similar to this one:
https://github.com/getredash/redash/blob/99c73aef2d925cac269dbfd2b763b6961f68992f/redash/query_runner/big_query.py#L16-L25

This will make the import safe and the data source disabled in case of the import failing.